components/busy-ui/busy-ui.html

```diff
@@ -7,9 +7,75 @@
         translate: -50% -50%;
         user-select: none;
         pointer-events: none;
+        text-align: center;
     }
+
+    .loading span {
+        width: 1rem;
+        height: 1rem;
+        background: var(--blue-s1);
+        border-radius: 100%;
+        display: inline-block;
+        -webkit-animation: dots 1.5s infinite ease-in-out both;
+        animation: dots 1.5s infinite ease-in-out both;
+    }
+
+    .loading span:nth-child(1) {
+        -webkit-animation-delay: -0.32s;
+        animation-delay: -0.32s;
+    }
+
+    .loading span:nth-child(2) {
+        -webkit-animation-delay: -0.16s;
+        animation-delay: -0.16s;
+    }
+
+    @-webkit-keyframes dots {
+        0%, 80%, 100% {
+            -webkit-transform: scale(0.3);
+            background-color: var(--blue-s1);
+        }
+        40% {
+            -webkit-transform: scale(0.8);
+            background-color: var(--blue)
+        }
+    }
+
+    @keyframes dots {
+        0%, 80%, 100% {
+            transform: scale(0.3);
+            background-color: var(--blue-s1);
+        }
+        40% {
+            transform: scale(0.8);
+            background-color: var(--blue);
+        }
+    }
+
+    #lblMessage {
+        text-align: center;
+        margin-top: 0.5rem;
+        margin-bottom: 0.5rem;
+        font-size: 1rem;
+        font-weight: bold;
+    }
+
+    #lblProgress {
+        text-align: center;
+        margin-bottom: 0.5rem;
+        font-weight: normal;
+        color: var(--grey-s1);
+        font-style: italic;
+        font-size: 1rem;
+        font-family: 'SourceSansPro', sans-serif;
```

Comment:
do we need this? can it be imported ?

Reply:
removed size and font-family, italic is a design requirement